src/lib.rs

```diff
@@ -13,6 +17,21 @@ pub enum Tee {
     Tdx,
 }
 
+/*
+ *  NOTE: Although not explicitly a part of the KBS attestation protocol, this
+ *        type can be handy when VMMs are looking to register an encrypted
+ *        workload with an attestation server. That way, when TEE-encrypted
+ *        guests are looking to attest, they can have a launch measurement to
+ *        compare with.
+ */
+#[derive(Serialize, Deserialize, Debug)]
+pub struct Register {
```

Comment:
I thought the purpose of this library was to define the interface between KBS and KBS/client. This part of the API seems unlikely to be standardized. Maybe it should be implemented in the KBS itself at least in a different file.

Reply:
It could very well be implemented in the attestation server. Will remove.